BitcoinApp/TransactionViewController/TransactionsTableViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  TransactionsTableViewController.swift
+//  BitcoinApp
+//
+//  Created by Zhassulan Aimukhambetov on 12/6/19.
+//  Copyright © 2019 Zhassulan Aimukhambetov. All rights reserved.
+//
+
+import UIKit
+
+class TransactionsTableViewController: UITableViewController {
+    
+    private let identifier = "toDetailTransaction"
+    var transactions: [Transaction] = []
```

Comment:
Pay attention to the Optionals.
Empty array ≠ nil. The empty array should work only for a real response with a zero number of responses. 
With respecting Optional you would be able to determine states like "not yet loaded" and "already loaded but got no response"

Reply:
Perfectly, it should be done with State Machine but it would be overkill in this case.